ui/tests/acceptance/cluster-test.js

```diff
@@ -9,7 +9,7 @@ const consoleComponent = create(consoleClass);
 
 const tokenWithPolicy = async function(name, policy) {
   await consoleComponent.runCommands([
-    `write sys/policies/acl/${name} policy=${policy}`,
+    `write sys/policies/acl/${name} policy=${btoa(policy)}`,
```

Comment:
Because of the changes in string handling, interpolation with multiple nested quotes didn't work properly any more for writing policies in tests. The work-around for this was to base64 encode the string prior to interpolation since vault's api supports writing policy in this way ✨